index.js

```diff
@@ -77,7 +77,7 @@ Taplytics.setTaplyticsNewSessionListener = (listener) => {
   Taplytics._setTaplyticsNewSessionListener()
```

Comment:
could change this to take two "listeners" representing success and failure

Reply:
Actually I don't think its that useful anymore to even do anything on error cause setTaplyticsNewSessionListener on the sdk side is a very trivial operation. The onError call is really on useful when you're calling startNewSession, since many things can potentially go wrong.